tests/org.eclipse.fordiac.ide.test.library/src/org/eclipse/fordiac/ide/test/library/Messages.java

```diff
@@ -0,0 +1,33 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Primetals Technologies Austria GmbH
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Patrick Aigner - initial API and implementation and/or initial documentation
+ *******************************************************************************/
+package org.eclipse.fordiac.ide.test.library;
+
+import org.eclipse.osgi.util.NLS;
+
+/** The Class Messages. */
+@SuppressWarnings("squid:S3008")
+public final class Messages extends NLS {
+	private static String BUNDLE_NAME = "org.eclipse.fordiac.ide.test.libraries.messages"; //$NON-NLS-1$
+
+	public static String Error;
+	public static String Library_Not_Found;
+
+	static {
+		// initialize resource bundle
+		NLS.initializeMessages(BUNDLE_NAME, Messages.class);
+	}
+
+	private Messages() {
+		// empty private constructor
+	}
+}
```

Comment:
This one is on me: my thought here was to just `import org.eclipse.fordiac.ide.gitlab.Messages;" to reuse the same translateable Messages